app/allocations/middleware/set-results.allocations.test.js

```diff
@@ -34,40 +38,61 @@ describe('Allocations middleware', function() {
 
     context('when service resolves', function() {
       beforeEach(async function() {
-        allocationService.getByDateAndLocation.resolves(mockActiveMoves)
+        allocationService.getByDateAndLocation
```

Comment:
These stubs can be updated, you're not longer actually calling this method within this middleware so they are no longer feel like the right thing to stub.

That way you can continue to use `calledOnceWithExactly` on each call too.